addons/website_event/tests/common.py

```diff
@@ -73,28 +73,28 @@ def setUpClass(cls):
         )
 
     def _get_menus(self):
-        return set(['Introduction', 'Location', 'Register'])
+        return set(['Introduction', 'Location', 'Register', 'Community'])
 
-    def _assert_website_menus(self, event, menu_entries=None):
+    def _assert_website_menus(self, event, menus_in, menus_out=None):
         self.assertTrue(event.menu_id)
 
-        if menu_entries is None:
-            menu_entries = self._get_menus()
+        if menus_in is None:
+            menus_in = list(self._get_menus())
```

Comment:
Looking at the new code, I'm not sure you ever set menus_in to None anywhere.

Reply:
I modified a test to do it :D . It was manually calling self._get_menus() .